src/components/FilterPanel/ZipCodeFilter.js

```diff
@@ -3,6 +3,9 @@ import FormGroup from "@material-ui/core/FormGroup";
 import FormLabel from "@material-ui/core/FormLabel";
 import TextField from "@material-ui/core/TextField";
 import { makeStyles } from "@material-ui/core/styles";
+import zipcodeData from "../../generated/ma-zips.json";
+// For performance, use a pared down list of Mass. zipcodes only (saves 374K or 60% of size!)
+// const zipcodeData = require("us-zips");
```

Comment:
Reordering the comment so that it comes after the line it applies to instead of before … does not make sense. Please don't do that; please retain the order as it was.

Reply:
Please also revert the change in `RadiusFilter.js`; sorry I didn't notice it there!